.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
What is the use of this line?

Reply:
The name of the GitHub event that triggers the workflow.

Reply:
Trigger the workflow on push or pull request, but only for the master branch